### DISCUSSION TITLE
Although this is not essentially an LXQt issue, does anyone have any idea how to fix the following issue in Ulauncher?

### DISCUSSION THREAD

**Discussion by Raytle** [lxqt/lxqt#2050] [Q&A]
Although this is not essentially an LXQt issue, does anyone have any idea how to fix the following issue in Ulauncher?

[ How can I prevent my large title fonts from overlapping my subtitle fonts? #759 ](https://github.com/Ulauncher/Ulauncher/issues/759)

**Comment by tsujan** [comment]
> ...not essentially an LXQt issue...

Absolutely not related to LXQt.

GTK apps can be used under LXQt but their issues aren't related to it.

**Reply by Raytle** [reply]
Thanks for letting me know. I appreciate that.
